03-specifying-sample-designs.Rmd

```diff
@@ -76,11 +78,11 @@ des_srs3 <- dat %>%
  as_survey_design(weights = wtvar, fpc = fpcvar)
 ```
 
-where `wtvar` is the variable for the weight on the data. Again, the fpc can be omitted if it is unnecessary because the frame is large.
+where `wtvar` is the variable for the weight on the data. Again, the FPC can be omitted if it is unnecessary because the frame is large.
 
 #### Example {-} 
 
-The {survey} package in R provides some example datasets that we will use throughout this chapter. Reading the documentation about these datasets provide detail on the variables. One of the example datasets we will use is from the Academic Performance Index (API). The API was a program administered by the California Department of Education, and the {survey} package includes a population file (frame) of all schools with at least 100 students and several different samples pulled from that data using different sampling methods. For this first example, we will use the `apisrs` dataset, which contains an SRS of 200 schools. For printing purposes, we create a new dataset called `apisrs_slim`, which sorts the data by the school district and school ID and subsets the data to only a few columns. The SRS sample data is illustrated below:
+The {survey} package in R provides some example datasets that we will use throughout this chapter. Reading the documentation about these datasets provide detail on the variables. One of the example datasets we will use is from the Academic Performance Index (API). The API was a program administered by the California Department of Education, and the {survey} package includes a population file (frame) of all schools with at least 100 students and several different samples pulled from that data using different sampling methods. For this first example, we will use the `apisrs` dataset, which contains an SRS of 200 schools. For printing purposes, we create a new dataset called `apisrs_slim`, which sorts the data by the school district and school ID and subsets the data to only a few columns. The SRS sample data is illustrated below: 
 
 ```{r}
```

Comment:
Let's create a "set up" section at the beginning of the chapter that has the libraries needed.  This is the first chapter where we have code, so we may need a little extra info around what each of the packages does?